test/MatrixSets/MatrixZonotope.jl

```diff
@@ -37,6 +37,22 @@ for N in @tN([Float64, Float32, Rational{Int}])
     @test isempty(generators(MZ3))
     @test isempty(indexvector(MZ3))
 
+    #scale
+    MZ_scaled = scale(2, MZ)
+    @test MZ_scaled isa MatrixZonotope{N}
+    @test MZ_scaled !== MZ  # not the same object
```

Comment:
```suggestion
```
This is redundant because the following tests forbid that (unless everything was 0).